### DISCUSSION TITLE
![[Markdown Note]] can't work in anki.....but I want see ![pdf] file in anki. Anyone know how to do?

### DISCUSSION THREAD

**Discussion by via-lactea7** [ObsidianToAnki/Obsidian_to_Anki#614] [Q&A]
plz help me 😆

**Comment by envico801** [comment]
Hi @via-lactea7, if I understand correctly, you are trying to make a link to a pdf file in a markdown note, but able to work in obsidian and anki, correct?

If this is the case, the reality is that anki is very limited in its capabilities, it only allows you to use HTML and basic Javascript, the closest thing to what you want to do is to use an <iframe> tag but this would only give you a link to open it in another application. E.g.

```html
<iframe style="width: 100%; height: 100%;" src="Test.pdf"></iframe>
```

Maybe using some addon you can do it, but it seems to me that you will have to modify the plugin to fit your needs in order to do it automatically.

And on the other hand, Obsidian-To-Anki (the plugin) does not allow to embed `.pdf` files. It will return this error by console: `Unsupported extension:  .pdf`

[https://github.com/ObsidianToAnki/Obsidian_to_Anki/blob/feb3db2708559bf386412ef6f8be00753faf7775/src/format.ts#L96](https://github.com/ObsidianToAnki/Obsidian_to_Anki/blob/feb3db2708559bf386412ef6f8be00753faf7775/src/format.ts#L96)

The allowed extensions are images and audio:

[https://github.com/ObsidianToAnki/Obsidian_to_Anki/blob/feb3db2708559bf386412ef6f8be00753faf7775/src/format.ts#L18-L19](https://github.com/ObsidianToAnki/Obsidian_to_Anki/blob/feb3db2708559bf386412ef6f8be00753faf7775/src/format.ts#L18-L19)
